modules/end-user-guide/pages/integrating-with-kubernetes.adoc

```diff
@@ -1,8 +1,9 @@
-:navtitle: Integrating with {orch-name}
+:navtitle: {orch-name} integrations
 :keywords: overview, integrating with {orch-name}
 :page-aliases: 
 
 [id="integrating-with-kubernetes_{context}"]
-= Integrating with {orch-name}
+= {orch-name} integrations
```

Comment:
Keep the gerund in the title. It is part of our requirements.

Reply:
ok, this was how the issue was titled see - https://github.com/eclipse/che/issues/21300

Reply:
`Integrating with Kubernetes` vs `Kubernetes integrations` 
those titles have different meanings and the current one is ambiguous imo

Reply:
reverted back to `Integrating with Kubernetes` as per request